Cargo.toml

```diff
@@ -36,6 +36,7 @@ base64 = "0.13" # Follows version used by rustls
 bytes = "1.0" # Follows version used by http
 chrono = { version = "0.4", features = ["serde"] }
 futures-util = { version = "0.3", optional = true, features = ["io"] }
+hex = "0.4"
```

Comment:
```suggestion
```

There is no need to add a dependency and remove the util code in a lint related PR.